package.json

```diff
@@ -110,6 +110,7 @@
     "karma-ng-html2js-preprocessor": "^0.1.2",
     "karma-phantomjs-launcher": "~0.2.0",
     "load-grunt-tasks": "^3.2.0",
+    "protractor": "git://github.com/angular/protractor.git#master",
```

Comment:
@ilanbiala I am seeing that travis is failing to run using node v0.10. it looks like the error is due to `node_modules/grunt-protractor-runner/node_modules/protractor/lib/configParser.js:102:29)` which seems to be an embedded protractor version..so, I suppose we need to wait until grunt-protractor-runner upgrades or we can use my [fork](https://github.com/jloveland/grunt-protractor-runner/tree/protractor-parse-error)..not sure what the best option is..